.gitignore

```diff
@@ -6,6 +6,7 @@
 *.log
 .nfs*
 build/
+scripts/linux-amd64/*
```

Comment:
Remove this line.

Reply:
I added this because, my local environment kept adding linux-amd64 scripts, but I will remove it